workloads/router-perf-v2/common.sh

```diff
@@ -147,7 +147,7 @@ gen_mb_config(){
     local port=443
   fi
   (echo "["
-  while read host; do
+  oc get route -n http-scale-${termination} --no-headers | awk '{print $2}'| while read host; do
```

Comment:
Why do you need this change?, are you using a different shell?

Reply:
We are running this in Jenkins. If you'd like we could share the existing failed runs or run a fresh job minus the changes proposed in this pr. Seems like @qiliRedHat was only able to succeed after the changes she proposing here were made.

Reply:
@rsevilla87 If I do not make this change, I met the problem as described in https://github.com/cloud-bulldozer/e2e-benchmarking/pull/201#issue-686565734 `Problem: Multiple hosts are put together as one host value.` Multiple hosts are passed to the while loop as one line so the host generated is like below. All hosts are generated to the same json object.
`"host": "<placeholder of host1> <placeholder of host2>",`.
Changing to the current implementation makes the hosts to be passed to the while loop as separated lines, so that each host get an json object created.

When you said `a different shell`, what shell do you originally use? As @kedark3 said I encountered this in our Jenkins. I did a test on different shells on the old and new codes as below. The new code survives all bash tested, but old code failed with `bash` and `sh`

I used this test.sh to simulate getting nodes instead of routes with old code.
```
while read node; do echo '{"node": "'${node}'"},'; done <<< $(oc get nodes --no-headers | awk '{print $1}')
```
Results of different shells: bash - Failed, zsh - Passed, ksh - Passed, sh - Failed.
```
 bash test.sh
+ bash test.sh
{"node": "ip-10-0-131-13.us-east-2.compute.internal ip-10-0-149-189.us-east-2.compute.internal ip-10-0-150-101.us-east-2.compute.internal ip-10-0-150-161.us-east-2.compute.internal ip-10-0-150-58.us-east-2.compute.internal ip-10-0-164-166.us-east-2.compute.internal ip-10-0-171-91.us-east-2.compute.internal ip-10-0-180-196.us-east-2.compute.internal ip-10-0-182-232.us-east-2.compute.internal ip-10-0-186-92.us-east-2.compute.internal ip-10-0-201-142.us-east-2.compute.internal ip-10-0-209-218.us-east-2.compute.internal ip-10-0-211-132.us-east-2.compute.internal ip-10-0-221-107.us-east-2.compute.internal"},
```

```
zsh test.sh
+ zsh test.sh
{"node": "ip-10-0-131-13.us-east-2.compute.internal"},
{"node": "ip-10-0-149-189.us-east-2.compute.internal"},
{"node": "ip-10-0-150-101.us-east-2.compute.internal"},
{"node": "ip-10-0-150-161.us-east-2.compute.internal"},
{"node": "ip-10-0-150-58.us-east-2.compute.internal"},
{"node": "ip-10-0-164-166.us-east-2.compute.internal"},
{"node": "ip-10-0-171-91.us-east-2.compute.internal"},
{"node": "ip-10-0-180-196.us-east-2.compute.internal"},
{"node": "ip-10-0-182-232.us-east-2.compute.internal"},
{"node": "ip-10-0-186-92.us-east-2.compute.internal"},
{"node": "ip-10-0-201-142.us-east-2.compute.internal"},
{"node": "ip-10-0-209-218.us-east-2.compute.internal"},
{"node": "ip-10-0-211-132.us-east-2.compute.internal"},
{"node": "ip-10-0-221-107.us-east-2.compute.internal"},
```

```
$ ksh test.sh
+ ksh test.sh
{"node": "ip-10-0-131-13.us-east-2.compute.internal"},
{"node": "ip-10-0-149-189.us-east-2.compute.internal"},
{"node": "ip-10-0-150-101.us-east-2.compute.internal"},
{"node": "ip-10-0-150-161.us-east-2.compute.internal"},
{"node": "ip-10-0-150-58.us-east-2.compute.internal"},
{"node": "ip-10-0-164-166.us-east-2.compute.internal"},
{"node": "ip-10-0-171-91.us-east-2.compute.internal"},
{"node": "ip-10-0-180-196.us-east-2.compute.internal"},
{"node": "ip-10-0-182-232.us-east-2.compute.internal"},
{"node": "ip-10-0-186-92.us-east-2.compute.internal"},
{"node": "ip-10-0-201-142.us-east-2.compute.internal"},
{"node": "ip-10-0-209-218.us-east-2.compute.internal"},
{"node": "ip-10-0-211-132.us-east-2.compute.internal"},
{"node": "ip-10-0-221-107.us-east-2.compute.internal"},
```

```
sh test.sh
+ sh test.sh
{"node": "ip-10-0-131-13.us-east-2.compute.internal ip-10-0-149-189.us-east-2.compute.internal ip-10-0-150-101.us-east-2.compute.internal ip-10-0-150-161.us-east-2.compute.internal ip-10-0-150-58.us-east-2.compute.internal ip-10-0-164-166.us-east-2.compute.internal ip-10-0-171-91.us-east-2.compute.internal ip-10-0-180-196.us-east-2.compute.internal ip-10-0-182-232.us-east-2.compute.internal ip-10-0-186-92.us-east-2.compute.internal ip-10-0-201-142.us-east-2.compute.internal ip-10-0-209-218.us-east-2.compute.internal ip-10-0-211-132.us-east-2.compute.internal ip-10-0-221-107.us-east-2.compute.internal"},
```

I used test_new.sh to simulate getting nodes (instead of routes) with the new code I updated in this pr.
```
oc get nodes --no-headers | awk '{print $1}' |while read node; do echo '{"node": "'${node}'"},'; done
```
All bashes is passed with this code
```
qili@qili-mac test % bash test_new.sh
{"node": "ip-10-0-131-13.us-east-2.compute.internal"},
{"node": "ip-10-0-149-189.us-east-2.compute.internal"},
{"node": "ip-10-0-150-101.us-east-2.compute.internal"},
{"node": "ip-10-0-150-161.us-east-2.compute.internal"},
{"node": "ip-10-0-150-58.us-east-2.compute.internal"},
{"node": "ip-10-0-164-166.us-east-2.compute.internal"},
{"node": "ip-10-0-171-91.us-east-2.compute.internal"},
{"node": "ip-10-0-180-196.us-east-2.compute.internal"},
{"node": "ip-10-0-182-232.us-east-2.compute.internal"},
{"node": "ip-10-0-186-92.us-east-2.compute.internal"},
{"node": "ip-10-0-201-142.us-east-2.compute.internal"},
{"node": "ip-10-0-209-218.us-east-2.compute.internal"},
{"node": "ip-10-0-211-132.us-east-2.compute.internal"},
{"node": "ip-10-0-221-107.us-east-2.compute.internal"},
qili@qili-mac test % zsh test_new.sh
{"node": "ip-10-0-131-13.us-east-2.compute.internal"},
{"node": "ip-10-0-149-189.us-east-2.compute.internal"},
{"node": "ip-10-0-150-101.us-east-2.compute.internal"},
{"node": "ip-10-0-150-161.us-east-2.compute.internal"},
{"node": "ip-10-0-150-58.us-east-2.compute.internal"},
{"node": "ip-10-0-164-166.us-east-2.compute.internal"},
{"node": "ip-10-0-171-91.us-east-2.compute.internal"},
{"node": "ip-10-0-180-196.us-east-2.compute.internal"},
{"node": "ip-10-0-182-232.us-east-2.compute.internal"},
{"node": "ip-10-0-186-92.us-east-2.compute.internal"},
{"node": "ip-10-0-201-142.us-east-2.compute.internal"},
{"node": "ip-10-0-209-218.us-east-2.compute.internal"},
{"node": "ip-10-0-211-132.us-east-2.compute.internal"},
{"node": "ip-10-0-221-107.us-east-2.compute.internal"},
qili@qili-mac test % ksh test_new.sh
{"node": "ip-10-0-131-13.us-east-2.compute.internal"},
{"node": "ip-10-0-149-189.us-east-2.compute.internal"},
{"node": "ip-10-0-150-101.us-east-2.compute.internal"},
{"node": "ip-10-0-150-161.us-east-2.compute.internal"},
{"node": "ip-10-0-150-58.us-east-2.compute.internal"},
{"node": "ip-10-0-164-166.us-east-2.compute.internal"},
{"node": "ip-10-0-171-91.us-east-2.compute.internal"},
{"node": "ip-10-0-180-196.us-east-2.compute.internal"},
{"node": "ip-10-0-182-232.us-east-2.compute.internal"},
{"node": "ip-10-0-186-92.us-east-2.compute.internal"},
{"node": "ip-10-0-201-142.us-east-2.compute.internal"},
{"node": "ip-10-0-209-218.us-east-2.compute.internal"},
{"node": "ip-10-0-211-132.us-east-2.compute.internal"},
{"node": "ip-10-0-221-107.us-east-2.compute.internal"},
qili@qili-mac test % sh test_new.sh
{"node": "ip-10-0-131-13.us-east-2.compute.internal"},
{"node": "ip-10-0-149-189.us-east-2.compute.internal"},
{"node": "ip-10-0-150-101.us-east-2.compute.internal"},
{"node": "ip-10-0-150-161.us-east-2.compute.internal"},
{"node": "ip-10-0-150-58.us-east-2.compute.internal"},
{"node": "ip-10-0-164-166.us-east-2.compute.internal"},
{"node": "ip-10-0-171-91.us-east-2.compute.internal"},
{"node": "ip-10-0-180-196.us-east-2.compute.internal"},
{"node": "ip-10-0-182-232.us-east-2.compute.internal"},
{"node": "ip-10-0-186-92.us-east-2.compute.internal"},
{"node": "ip-10-0-201-142.us-east-2.compute.internal"},
{"node": "ip-10-0-209-218.us-east-2.compute.internal"},
{"node": "ip-10-0-211-132.us-east-2.compute.internal"},
{"node": "ip-10-0-221-107.us-east-2.compute.internal"},
```